### PR TITLE
chore: update major/minor release issue template

### PR DESCRIPTION
**Pull request by sugat009**
# Description

Updated the major/minor release issue template.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update_major_release_issue_template/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update_major_release_issue_template/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update_major_release_issue_template/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/z_release_major.md [hunk 1]
```diff
@@ -74,5 +75,4 @@ We’ve also implemented loads of other improvements and fixed a heap of bugs.
 
 ```
 - [ ] Add one last update to the #product-team Slack channel and use the thread to lead an internal release retrospective covering what went well and areas to improve for next time.
-- [ ] Go to the [Issues tab](https://github.com/medic/cht-core/issues) and filter the issues with `is:issue label:"Affects: 4.x.x" ` , replace `4.x.x` with the previous version number. Add any open "known issues" from the prior release that were not fixed in this release. Done by adding the correct `Affects: 4.x.x` label.
```
Comment (mrjones-plip) [inline .github/ISSUE_TEMPLATE/z_release_major.md:77]: I think we want keep this line - please add back

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Looks good - thanks!

I think we want to keep the one line about re-assigning unfixed known issues though, yeah?

File: .github/ISSUE_TEMPLATE/z_release_major.md (hunk 1 above)
Comment (mrjones-plip) [reply .github/ISSUE_TEMPLATE/z_release_major.md:77]: oh - so you did - sorry I missed that !

**Review by mrjones-plip — APPROVED** [review]
LTGM :ship:

File: .github/ISSUE_TEMPLATE/z_release_major.md (hunk 1 above)
Comment (garethbowen) [reply .github/ISSUE_TEMPLATE/z_release_major.md:77]: We definitely still need the line - has it been moved above?

**Pull request merged**